soc/xtensa/intel_adsp/tools/cavstool.py

```diff
@@ -551,6 +555,18 @@ def ipc_command(data, ext_data):
         for i in range(0, 256):
             buf[i] = i
         hda_streams[stream_id].write(buf)
+    elif data == 12: # HDA PRINT
```

Comment:
Would prefer to see the protocol handling, which is a test thing very unlikely to be used by production code (SOF has its own much more elaborate IPC protocol) cleanly separated from the driver here in a different patch.